client/stripe-link/index.js

```diff
@@ -4,19 +4,22 @@ const showLinkButton = ( linkAutofill ) => {
 		const linkButtonTop =
 			jQuery( '#billing_email' ).position().top +
 			( jQuery( '#billing_email' ).outerHeight() - 40 ) / 2;
-		jQuery( '.wcpay-stripelink-modal-trigger' ).show();
-		jQuery( '.wcpay-stripelink-modal-trigger' ).css(
+		jQuery( '.stripe-gateway-stripelink-modal-trigger' ).show();
+		jQuery( '.stripe-gateway-stripelink-modal-trigger' ).css(
 			'top',
 			linkButtonTop + 'px'
 		);
 	}
 
 	// Handle StripeLink button click.
-	jQuery( '.wcpay-stripelink-modal-trigger' ).on( 'click', ( event ) => {
-		event.preventDefault();
-		// Trigger modal.
-		linkAutofill.launch( { email: jQuery( '#billing_email' ).val() } );
-	} );
+	jQuery( '.stripe-gateway-stripelink-modal-trigger' ).on(
+		'click',
+		( event ) => {
+			event.preventDefault();
+			// Trigger modal.
+			linkAutofill.launch( { email: jQuery( '#billing_email' ).val() } );
+		}
+	);
```

Comment:
I think I found a bug here. In the first time we click, the Link modal shows up. If we close the modal and try to click in the button again, the modal is not displayed.

Reply:
Good catch @cesarcosta99 ! Indeed, if you close the modal and try to open it again, the modal is not displayed. This is on Stripe side. I will ask Stripe to check this behavior.
Thank you!

Reply:
Gotcha, then it's fine to go ignore that for now, but it'd be nice if you could create an issue to track this bug.